src/main/java/me/mrCookieSlime/Slimefun/androids/ProgrammableAndroid.java

```diff
@@ -499,7 +500,11 @@ private void mine(Block b, BlockMenu menu, Block block) {
 		Collection<ItemStack> drops = block.getDrops();
 		if (!blockblacklist.contains(block.getType()) && !drops.isEmpty() && SlimefunPlugin.getProtectionManager().hasPermission(Bukkit.getOfflinePlayer(UUID.fromString(BlockStorage.getLocationInfo(b.getLocation(), "owner"))), block.getLocation(), ProtectableAction.BREAK_BLOCK)) {
 			String item = BlockStorage.checkID(block);
-
+			AndroidMineEvent event = new AndroidMineEvent(block);
+			Bukkit.getPluginManager().callEvent(event);
```

Comment:
Shouldn't this also be called in the "movedig" function?

Reply:
Oh right!